pkg/scheduler/core/util.go

```diff
@@ -72,8 +72,9 @@ func calAvailableReplicas(clusters []*clusterv1alpha1.Cluster, spec *workv1alpha
 	estimators := estimatorclient.GetReplicaEstimators()
 	ctx := context.WithValue(context.TODO(), util.ContextKeyObject,
 		fmt.Sprintf("kind=%s, name=%s/%s", spec.Resource.Kind, spec.Resource.Namespace, spec.Resource.Name))
-	for _, estimator := range estimators {
+	for name, estimator := range estimators {
 		res, err := estimator.MaxAvailableReplicas(ctx, clusters, spec.ReplicaRequirements)
+		klog.V(5).Infof("MaxAvailableReplica scores calculated by estimator %s: %v", name, availableTargetClusters)
```

Comment:
It would be great if this log carries the referring workload, like:
https://github.com/karmada-io/karmada/blob/957e50b2cf55c2bdd8eeabafcc1cde3da480f686/pkg/scheduler/core/util.go#L66

otherwise, we can't figure out which workload/resource the log belongs to.

Reply:
Updated log shows:
```
I0604 15:56:07.114751       1 util.go:78] MaxAvailableReplica scores calculated by estimator general-estimator for workload(kind=FlinkDeployment, test-namespace/mszacillo-karmada-test-2): [{cluster-A 8} {cluster-B 8}]
```